Update responsive-domain-search.js

SpookyKipper opened the pull request:
It seems that it currently targets the Addon, Sub, and parked domains page and makes the page only has the domain search form left.

This fixes it

SpookyKipper committed:
Update responsive-domain-search.js

aa22dev commented:
I don't think that there is a need for a check to implement the code only on the homepage. This code can be and I think should be implemented for all pages containing non-responsive fields.

SpookyKipper commented:
But the current code completely nukes the Addon, sub, and parked domains page (for me)

4yx commented:
Per [README](https://github.com/WybeNetwork/VistaPanel-Customizations/tree/master/responsive-domain-search#where-should-i-put-it):
> Put it in the Header Advert Area

Putting the code in the header advert area is meant to work as the solution for this issue, as the header advert area is supposed to be loaded only on the first screen. Are you facing this issue whilst having it there?

4yx committed:
Responsive Domain Search: Style updates

Update description to mention SpookyKipper as the last one to update the script, and make a minor stylistic change to the code fetching the option.

Also, add some comments explaining the code. Shouldn't be necessary in this era, but it looks nice regardless.
